lib/pom.xml

```diff
@@ -14,9 +14,8 @@
 
     <properties>
         <version.model>0.1.0-SNAPSHOT</version.model>
-
-        <version.core.lightblue>1.11.0</version.core.lightblue>
-        <version.mongo.lightblue>1.11.0</version.mongo.lightblue>
+        <version.core.lightblue>1.12.0-SNAPSHOT</version.core.lightblue>
+        <version.mongo.lightblue>1.12.0-SNAPSHOT</version.mongo.lightblue>
```

Comment:
It doesn't depend on it, but I didn't have 11 snapshot, so I upgraded. I
can downgrade if you want,

On Thu, Mar 17, 2016 at 3:07 PM, Alec Henninger <notifications@github.com>
wrote:

> In lib/pom.xml
> <https://github.com/esbtools/lightblue-notification-hook/pull/10#discussion_r56577813>
> :
>
> > @@ -14,9 +14,8 @@
> >
> >      <properties>
> >          <version.model>0.1.0-SNAPSHOT</version.model>
> > -
> > -        <version.core.lightblue>1.11.0</version.core.lightblue>
> > -        <version.mongo.lightblue>1.11.0</version.mongo.lightblue>
> > +        <version.core.lightblue>1.12.0-SNAPSHOT</version.core.lightblue>
> > +        <version.mongo.lightblue>1.12.0-SNAPSHOT</version.mongo.lightblue>
>
> Does this change depend on 1.12.0-SNAPSHOT features? What's the release
> timeline for 1.12.0? I'm worried a bit with the lightblue versions that we
> may get out of sync when we go to release the hook. I'm not really sure
> what lightblue version we should be using when we have dependencies on
> lightblue components so if you have some general guidelines to share that
> would be appreciated.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/esbtools/lightblue-notification-hook/pull/10/files/0507ae1f8980cdc0d11725e3066130cbf63a5604#r56577813>
>